backend/python-functions/deploy.py

```diff
@@ -13,14 +13,24 @@
 if args.env == "development":
     cloud_fn_name += "_development"
 
+trigger_options = {
+    "ingest_covid_case_data": [
+        "--trigger-resource", "c19-backend-covid-case-data",
+        "--trigger-event", "google.storage.object.finalize",
+    ],
+    "calculate_rt" : [
+        "--trigger-http",
+    ]
+}
```

Comment:
I'm not sure how generic this deployment script was intended to be, but I've made it less generic with this function-specific trigger configuration.  If this feels egregious, I can work to update this script to take in more parameters to represent what I'm currently capturing in this dictionary.

I've tested the following commands and they work as expected:

```sh
python deploy.py calculate_rt development
python deploy.py ingest_covid_case_data production
```